shell/src/main/java/alluxio/cli/fs/command/StatCommand.java

```diff
@@ -61,6 +63,11 @@ public Options getOptions() {
             .hasArg()
             .desc("format")
             .build()
+    ).addOption(
+        Option.builder("fileId")
```

Comment:
Make it a `longOpt` `--file-id` to align with the convention in other commands?

Reply:
done